src/TuxArchitectStores.js

```diff
@@ -62,34 +62,64 @@ ArchitectChain.prototype.itOutputs = function (output) {
   return this;
 };
 
-//_getRegisterId FUNCTION: internal function,for getting the store __registerId__ from either the passed in store or the store mapped to the passed in input
+//_getRegisterId FUNCTION: internal function, for getting the store or set of stores' __registerId__ or set of __registerId__'s from either the passed in store or the store mapped to the passed in input
```

Comment:
Change comment to ...from either the passed in store or the store/set of stores mapped to the passed in input.